### DISCUSSION TITLE
v10 multiSelectionKeyCode?

### DISCUSSION THREAD

**Discussion by nousernames2** [xyflow/xyflow#2208] [Q&A]
I can't setup a custom multiSelectionKeyCode in v10. I'm trying to use the control and shift keys.

I've tried "ControlLeft", "Control", even 17 and "Ctrl" but none of these work. For shift I've tried "ShiftLeft", "Shift" and 16. 

None of which are working.

**Comment by moklick** [comment]
Currently "Control" doesn't work, because it triggers a right click that we don't prevent (there is an open issue for that #2203). "Shift" works but it is used for the `selectionKeyCode`. You need to set the `selectionKeyCode` to another key to be able to choose "Shift" as you `multiSelectionKeyCode`.

**Reply by nousernames2** [reply]
Makes sense. Thanks
